### PR TITLE
fix: Docker and Docker Compose

### PR DESCRIPTION
**Pull request by roelofb**
### Problem
Docker and Docker Compose did not work

### Solution
Simply want to type `docker-compose up` and have the platform up and running, including database server

### Proposed of changes
Moved Dockerfile to root to be able to build project dependencies, reworked docker-compose to run mssql also.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,24 @@ version: '3'
...
+    environment:
+      ConnectionStrings__VirtoCommerce: "Data Source=mssql;Initial Catalog=VirtoCommerce3;Persist Security Info=True;User ID=virto;Password=virto;MultipleActiveResultSets=True;Connect Timeout=30"
+      ASPNETCORE_ENVIRONMENT: Development
+    build: .
+    restart: on-failure
+    depends_on:
+      - mssql
+    ports:
+      - "20654:80"
+
```
Comment (mvktsk) [inline docker-compose.yml]: Plaease add 
```
networks:
      - virto
```

File: docker-compose.yml [hunk 2]
```diff
@@ -2,7 +2,24 @@ version: '3'
...
+    image: mcr.microsoft.com/mssql/server:2019-GA-ubuntu-16.04
+    container_name: mssql
+    command: /bin/bash -c '/usr/src/app/mssql-docker-command.sh'
+    ports:
+      - 1433:1433
+    environment:
+      SA_PASSWORD: "Virto3.0"
+      ACCEPT_EULA: "Y"
+    volumes:
+      - ./db:/usr/src/app
```
Comment (mvktsk) [inline docker-compose.yml]: Please add 
```
networks:
      - virto
```

File: link_mods.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# This script builds, publishes and symlinks all Virto modules.
```
Comment (mvktsk) [inline link_mods.sh]: Could not find where link_mods.sh was used

File: gh_mods.sh [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# This script clones or pulls all Virto 3 modules.
```
Comment (mvktsk) [inline gh_mods.sh]: Could not find where gh_mods.sh was used

**Comment by yecli** [comment]
Too long no activity

**Pull request closed without merging**